pkg/test/framework/components/istio/operator.go

```diff
@@ -480,7 +480,7 @@ func installControlPlaneCluster(i *operatorComponent, cfg Config, cluster resour
 		}
 	}
 
-	if i.environment.IsConfigCluster(cluster) {
+	if i.environment.IsMulticluster() && i.environment.IsConfigCluster(cluster) {
```

Comment:
Without this change, "eastwest-gateway" seems to be installed even for the single cluster scenarios.
cc: @ramaraochavali

Reply:
Its used for VMs as well

Reply:
@howardjohn , sorry, I am not completely following.
Is the change here ok to keep, or it will break VMs?

Reply:
Found an answer in the results of the integration tests.
Reverting this change fixes the issue.

But, it seems that eastwest-gateway is even being installed for the test added in this PR. @howardjohn , do you know if there's a way to somehow disable it while initializing a NewSuite() - https://github.com/istio/istio/pull/27861/files#diff-b8125949e889b8b57cb90c1f16049b984e751c4cfac683ca1915224ca24ff65aR63